app/helpers/alerts_helper.rb

```diff
@@ -1,3 +1,11 @@
 # frozen_string_literal: true
 module AlertsHelper
+  def alert_display
+    if alert.resolvable? == true && alert.resolved == true then
```

Comment:
@arun1595, @mz99  - the story says this: ``` Instead of showing either 'N/A' or the resolve link, that column should show a check mark or similar indicator if the alert is resolved, and should show nothing at all if it is unresolvable.```

Also, I think the logic is constrained by this: `resolvable?` will return `true` _if_ the item is, in fact, resolvable _and_ has not been resolved - otherwise `resolvable?` returns `false`.

So, Arun's proposed logic needs to reflect these items.